build-container.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
```

Comment:
it looks this script is almost the same except some parameters than the previous one so better to provide a flag in a common script ?

Reply:
Moved to a common script